go/obscuronode/enclave/enclaverunner/cli.go

```diff
@@ -56,6 +56,7 @@ func ParseConfig() config.EnclaveConfig {
 		return fileBasedConfig(*configPath)
 	}
 
+	fmt.Printf("addr: %s\n", *erc20ContractAddrs)
```

Comment:
Was this left in on purpose? I think any logging should be done somewhere more centralised (e.g. in the sim, or in the enclave start-up).

Reply:
it was for debugging. removed